cylc/flow/etc/examples/cycle-over-irregular-dates/index.rst

```diff
@@ -0,0 +1,132 @@
+Irregular Cycling
+-----------------
+
+We typically schedule tasks on regular intervals, e.g. ``P1D`` (every day) or
+``PT1H`` (every hour), however, sometimes our intervals are irregular.
+
+:ref:`user_guide.scheduling.exclusions` can be used to "subtract" dates or
+entire recurrences e.g:
+
+``PT1H!PT6H``
+   Every hour, except every six hours.
+``PT1H!(T00, T12)``
+   Every hour, except at 00:00 and 12:00.
+
+However, sometimes we want to schedule tasks on completely irregular intervals
+or at arbitrary dates. E.g, when working on case studies, you might have a list
+or range of arbitrary dates to work with.
+
+
+.. rubric:: Simple Example
+
+.. admonition:: Get a copy of this example
+   :class: hint
+
+   .. code-block:: console
+
+      $ cylc get-resources examples/cycle-over-irregular-dates/simple
+
+This example uses :ref:`Jinja` to define the list of dates and write out a
+scheduling section for each.
+
+.. literalinclude:: simple/flow.cylc
+   :language: cylc
+
+.. tip::
+
+   You can see the result of this Jinja2 code by running the ``cylc view -p``
```

Comment:
any reason why `-p` (which expands everything), not `-j` which just expands the jinja2?

Reply:
I think `-p` intelligently decides whether to use Jinja2 or EmPy pre-processing which makes it a little more generic / flexible.

Reply:
`-p` and `-j` are otherwise equivalent:

https://github.com/cylc/cylc-flow/blob/a722b265ad0bd68bc5366a8a90b1dbc76b9cd282/cylc/flow/scripts/view.py#L126-L143